data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report2_row_suppression.py

```diff
@@ -29,12 +30,15 @@
 # Parameters
 project_id = ""
 deid_cdr = ""
-com_cdr =""
-run_as = ""
-# -
+com_cdr ="" 
+run_as=""
 
-# df will have a summary in the end
-df = pd.DataFrame(columns = ['query', 'result']) 
+# +
+# Parameters
+#project_id = "aou-res-curation-prod"
```

Comment:
This parameter should be removed, we do not want our dataset and project names in the git history.